stdlib/public/core/Availability.swift

```diff
@@ -45,3 +45,57 @@ public func _stdlib_isOSVersionAtLeast(
   return false._value
 #endif
 }
+
+@inlinable
+internal var _stdlibCurrentVersion: (Int, Int) {
+  // On the master branch, increment the first number.
+  // On release branches, increment the second number.
+  return (1001, 0)
+}
+
+/// Returns the version number for the Swift Standard Library that the code
+/// calling this was originally compiled with.
+///
+/// The version number is an arbitrary pair of integers, with lexicographical
+/// ordering. Version numbers of (logically) successive stdlib releases form a
+/// monotonically increasing sequence; i.e., versions should not decrease, but
+/// they are allowed to stay the same.
+///
+/// The two integer components are not intended to correspond to major or minor
+/// versions in a semantic versioning scheme. Neither do they correlate with
+/// toolchain or OS version numbers.
+public var _stdlibStaticVersion: (Int, Int) {
+  @_alwaysEmitIntoClient
```

Comment:
Why do we want back deployment?

Reply:
In this case, `@_alwaysEmitIntoClient` is not really here to allow back-deployment -- it's to get rid of the symbol in libswiftCore. This is akin to a header-only `#define` constant; the static version number should never ever be linked in from the currently running stdlib.

Reply:
AEIC gets rid of the symbol for the getter. You're just swapping it for `_stdlibOpaqueVersion` in exchange for more stuff in our binary compatibility story. How is that better?

Reply:
I still don't get it -- `_stdlibStaticVersion` is a compile-time constant for the version of the stdlib that the code was compiled with. By definition, it is public API that must not ever become an ABI symbol in libswiftCore. `@_alwaysEmitIntoClient` is perfect for it.

`_stdlibOpaqueVersion` is different -- it returns the version of the stdlib that is loaded at runtime, so it must be non-inlinable, with an availability.

Reply:
Oh wait, I see what you mean -- we're swapping it for `_stdlibCurrentVersion`, not `_stdlibOpaqueVersion`. Yeah, that needs to be fixed; I apparently forgot to get rid of it.